specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/stable/2024-04-03/examples/MsixImage_Expand_Post.json

```diff
@@ -14,7 +14,7 @@
         "value": [
           {
             "name": "hostpool1/expandmsiximage",
-            "id": "/subscriptions/daefabc0-95b4-48b3-b645-8a753a63c4fa/resourcegroups/resourceGroup1/providers/Microsoft.DesktopVirtualization/hostPools/hostpool1/expandMsixImage",
+            "id": "/subscriptions/daefabc0-95b4-48b3-b645-8a753a63c4fa/resourcegroups/resourceGroup1/providers/Microsoft.DesktopVirtualization/hostPools/hostpool1/expandMsixImage/msixImage",
```

Comment:
>msixImage [](http://example.com/codeflow?start=179&length=9)

Is this a new resource type? I couldnt find this path in the json.

Reply:
It is not a new resource type, this path returns a list of expandMsixImage resources "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DesktopVirtualization/hostPools/{hostPoolName}/expandMsixImage"

Reply:
I see that the id in resource below is updated to an id for msixImageURI. 

Where is the type behind this armid defined (the id on which this comment rests) - "/subscriptions/daefabc0-95b4-48b3-b645-8a753a63c4fa/resourcegroups/resourceGroup1/providers/Microsoft.DesktopVirtualization/hostPools/hostpool1/expandMsixImage/msixImage" ? 
Does this need to be updated as well?

Reply:
I updated it to match the example in 2024-04-08-preview which was created by the developer who worked on this feature. Since this response is a list of expandMsixImage objects it makes sense for them to have different ids.